caproto/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.1" %}
+
+package:
+    name: caproto
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NSLS-II/caproto
+    git_rev: v{{ version }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    skip: True  # [py2k]
+
+requirements:
+    build:
+        - python
+
+    run:
+        - curio
+        - netifaces
```

Comment:
What is `netifaces`?

Reply:
:man_shrugging: I copied the nsls-ii recipe wholesale and changed the version to latest